lib/charms/layer/nodejs.py

```diff
@@ -12,11 +12,10 @@ def node_dist_dir():
     Returns:
     Absolute string of node application directory
     """
-    storage_id = hookenv.storage_list('app')[0]
-    return hookenv.storage_get('location', storage_id)
```

Comment:
Is no one else using the nodejs layer? How will removing the current storage implementation affect other users?